third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -187,7 +187,7 @@ func resourceSqlDatabaseInstance() *schema.Resource {
 									"private_network": &schema.Schema{
 										Type:             schema.TypeString,
 										Optional:         true,
-										ValidateFunc:     validateRegexp(privateNetworkLinkRegex),
+										ValidateFunc:     orEmpty(validateRegexp(privateNetworkLinkRegex)),
```

Comment:
What does orEmpty do?  return true if the input is empty, or if the function returns true?

Reply:
Exactly that!